src/ls.js

```diff
@@ -1,5 +1,6 @@
 var path = require('path');
 var fs = require('fs');
+var glob = require('glob');
```

Comment:
Is it convenient to move all the `glob` stuff into `common.js`? That would be more shell-like, since a shell handles all the globbing itself (not the specific utilities).